tests/UtestPlatformTest.cpp

```diff
@@ -30,6 +30,31 @@
 #include "CppUTest/TestTestingFixture.h"
 #include "CppUTest/PlatformSpecificFunctions.h"
 
+TEST_GROUP(UTestPlatformsTest)
+{
+};
+
+TEST(UTestPlatformsTest, PlatformSpecificFileFunctionsDontCrash)
+{
+    PlatformSpecificFile file;
+    file = PlatformSpecificFOpen("/dev/null", "rw");
+    PlatformSpecificFPuts("Hello", file);
+    PlatformSpecificFClose(file);
+}
+
+TEST(UTestPlatformsTest, GetPlatformSpecificTimeStringIsCovered)
+{
+    GetPlatformSpecificTimeString();
```

Comment:
I see your point. But then, it's about as much of a test as the IGNORE_TEST() one I was complaining about a while back ;-). Also, it proves that it doesn't crash :-/